### ISSUE TITLE
Remove mentions of "draft" from the finalized note

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#582]
The published WCAG2ICT Note has 2 mentions of the document being a draft that need changing:

- [ ] [SOTD](https://www.w3.org/TR/wcag2ict/#sotd) section says, "This is a draft document and may be updated, replaced or obsoleted by other documents at any time. It is inappropriate to cite this document as other than work in progress." These statements need to be deleted.
- [ ] [Comparison with the 2013 WCAG2ICT Note](https://www.w3.org/TR/wcag2ict/#comparison-with-the-2013-wcag2ict-note) section, the next to last last sentence starts with, "In this draft, most" which should instead be, "Most".

**Comment by daniel-montalvo** [comment]
The above commit takes care of our instances of "draft" that should not be there.

Unfortunately, the 'draft" in the status of this document section comes from respec and cannot fix it straight away. I'll check whether this is done in purpose or there is a respec bug.
